glossary.md

```diff
@@ -40,20 +44,70 @@ A [Region](#regionpeerraft-group) is logically divided into several small ranges
 
+### Common table expression (CTE)
+
+A Common Table Expression (CTE) is a temporary result set that can be referred to multiple times within a SQL statement to improve the statement readability and execution efficiency. You can apply the `WITH` statement to use CTE.
+
+Common Table Expressions can be classified into two types: non-recursive CTE and recursive CTE.
+
+For details, see [Common table expression (CTE)](/develop/dev-guide-use-common-table-expression.md).
+
 ### Continuous Profiling
 
 Introduced in TiDB 5.3.0, Continuous Profiling is a way to observe resource overhead at the system call level. With the support of Continuous Profiling, TiDB provides performance insight as clear as directly looking into the database source code, and helps R&D and operation and maintenance personnel to locate the root cause of performance problems using a flame graph. For details, see [TiDB Dashboard Instance Profiling - Continuous Profiling](/dashboard/continuous-profiling.md).
 
+### Coprocessor
+
+A coprocessing mechanism that shares the computation workload with TiDB. It is located in the storage layer (TiKV or TiFlash) and collaboratively processes computations pushed down from TiDB on a per-region basis.
+
 ## D
 
+### Dumpling
+
+Dumpling is a data export tool for exporting data stored in TiDB/MySQL as SQL or CSV data files and can be used to make a logical full backup or export. Dumpling also supports exporting data to Amazon S3.
+
+For details, see [Use Dumpling to Export Data](/dumpling-overview.md).
+
 ### Dynamic Pruning
 
 Dynamic pruning mode is one of the modes that TiDB accesses partitioned tables. In dynamic pruning mode, each operator supports direct access to multiple partitions. Therefore, TiDB no longer uses Union. Omitting the Union operation can improve the execution efficiency and avoid the problem of Union concurrent execution.
 
+## E
+
+### Expression index
+
+The expression index is a type of special index that can be created on an expression. Once an expression index is created, TiDB can use the index for the expression-based query, which significantly improves the query performance.
+
+For details, see [CREATE INDEX - Expression index](/sql-statements/sql-statement-create-index.md#expression-index).
+
+## G
+
+### GC (Garbage Collection)
+
+Garbage collection (GC) is the memory resource management mechanism in TiDB. When old data in dynamic memory is no longer needed, it is cleaned up to free up memory.
+
+For details, see [GC Overview](/garbage-collection-overview.md).
+
+## H
+
+### Hotspot
+
+Hotspot refers to the phenomenon where the read and write workload of TiKV is concentrated on one or several regions or nodes, which might cause performance bottlenecks and prevent optimal performance. To solve hotspot issues, refer to [Troubleshoot Hotspot Issues](/troubleshoot-hot-spot-issues.md).
```

Comment:
```suggestion
Hotspot refers to the phenomenon where the read and/or write workload of TiKV is concentrated on one or several regions or nodes, which might cause performance bottlenecks and prevent optimal performance. To solve hotspot issues, refer to [Troubleshoot Hotspot Issues](/troubleshoot-hot-spot-issues.md).
```

Might also be good to call regions "data regions" instead to make it clear we're not talking about geographical regions (e.g [this](https://aws.amazon.com/about-aws/global-infrastructure/regions_az/)  )

Reply:
That's a good point. While we don't often refer to both terms simultaneously in our documentation, it can be confusing when we do. Currently, we capitalize "Region" for TiDB-specific regions and use lowercase "region" for cloud service provider regions. Using "data region" would enhance accuracy and improve clarity. "data region" would make it more accurate and easier to follow. I'll see if we can gradually transition to "data region" in other docs.